tools/speech_data_explorer/data_explorer.py

```diff
@@ -81,6 +81,7 @@ def parse_args():
     parser.add_argument(
         'manifest', help='path to JSON manifest file',
     )
+    parser.add_argument('--vocab', help='optional vocabulary to highlight OOV words')
```

Comment:
Let's note that --vocab points to a tab seperated file or a single word per line